test/utils/test_cost_funcs.py

```diff
@@ -0,0 +1,120 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test pulse scaling."""
+
+import unittest
+
+from mapomatic import deflate_circuit, evaluate_layouts, matching_layouts
+import numpy as np
+
+from qiskit import transpile
+from qiskit.circuit import Parameter, QuantumCircuit
+from qiskit.providers.fake_provider import FakeWashington
+
+from qiskit_research.utils.gates import SECRGate
+from qiskit_research.utils.convenience import (
+    add_dynamical_decoupling,
+    attach_cr_pulses,
+)
+from qiskit_research.utils.cost_funcs import cost_func_scaled_cr
+
+
+class TestScaledCostFuncs(unittest.TestCase):
+    """Test cost functions for scaled pulses."""
+
+    def test_cost_func_rzx(self):
+        """Test cost function for RZX"""
+        backend = FakeWashington()
+        rng = np.random.default_rng()
```

Comment:
I may not have been consistent about this in the past, but the tests should be seeded. For example,
```python
rng = np.random.default_rng(9370)
```

Reply:
Done!